server/src/test/java/org/opensearch/extensions/ExtensionsOrchestratorTests.java

```diff
@@ -260,12 +269,14 @@ public void testExtensionsDiscovery() throws Exception {
                     "fakeClass2",
                     new ArrayList<String>(),
                     true
-                )
+                ),
+                List.of(new ExtensionDependency("uniqueId0", Version.fromString("2.0.0")))
```

Comment:
since you have defined `expectedDependency` above you can just use it here (e.g., `List.of(expectedDependency)`